lib/internal/Magento/Framework/View/Test/Unit/Layout/Argument/Interpreter/ObjectTest.php

```diff
@@ -29,12 +30,36 @@ class ObjectTest extends \PHPUnit\Framework\TestCase
     protected function setUp()
     {
         $this->_objectManager = $this->createMock(\Magento\Framework\ObjectManagerInterface::class);
-        $this->_model = new DataObject($this->_objectManager, self::EXPECTED_CLASS);
+        $this->_model = new DataObject($this->_objectManager, self::EXPECTED_CLASS, new BooleanUtils());
```

Comment:
You should not rely on the implementation of another class when writing a unit test. So please mock BooleanUtils class as well.

Reply:
In general you made it right

Reply:
Fixed in 6bdb7db